cli/medperf/tests/test_utils.py

```diff
@@ -153,7 +153,7 @@ def test_cleanup_removes_only_invalid_datasets(mocker, datasets):
 
 
 @pytest.mark.parametrize("path", ["path/to/uids", "~/.medperf/cubes/"])
-@pytest.mark.parametrize("datasets", rand_l(1, 1000, 2), indirect=True)
+@pytest.mark.parametrize("datasets", [4, 287], indirect=True)
```

Comment:
There is still randomness when executing the `datasets` fixture defined on line 33. These numbers here are used to randomly sample UIDs in the pytest fixture.

Reply:
Good point! I didn't look at the fixtures. I've removed those calls as well